functest/helloworld_test.go

```diff
@@ -229,7 +229,31 @@ func TestCallHelloWorld(t *testing.T) {
 	//      requests should be less or equal to result count of registered requests
 	a.LessOrEqual(100, count)
 }
+func (i *HelloWorldInstance) PulseNumber(ctx context.Context) (int, error) {
```

Comment:
Don't interrupt test funcs and hw methods

Reply:
Done